### PR TITLE
chore(): release @angular/fire 5.0 🎉🔥🕺

### PR DESCRIPTION
**Pull request by jamesdaniels**
It's happening!

### CODE REVIEW

**Comment by jamesdaniels** [comment]
FYI [angular-fire-5.0.0.tgz](https://drive.google.com/file/d/1X7sxE3X65xdbYxQt0bisuOT8FMYaO_o0/view?usp=sharing)

**Comment by itsthesteve** [comment]
So happy to see it come under the `@angular` scope!!

**Comment by Chan4077** [comment]
So will the `angularfire2` package be marked as deprecated?

This can be a breaking change for old projects that still rely on this (soon to be) old package.

**Comment by davideast** [comment]
@Chan4077 We will do a final release of 5.0.0 for the `angularfire2` package. However, we will mark it as deprecated and move folks over to `@angular/fire`. Do you have any other ideas to help ease the move?

**Comment by Chan4077** [comment]
Schematics maybe?

**Comment by davideast** [comment]
@Chan4077 That's a fantastic idea

File: tools/build.js [hunk 1]
```diff
@@ -24,14 +24,14 @@ const GLOBALS = {
...
   'firebase/storage': 'firebase',
-  'angularfire2': 'angularfire2',
-  'angularfire2/auth': 'angularfire2.auth',
-  'angularfire2/database': 'angularfire2.database',
-  'angularfire2/database-deprecated': 'angularfire2.database_deprecated',
-  'angularfire2/firestore': 'angularfire2.firestore',
-  'angularfire2/functions': 'angularfire2.functions',
-  'angularfire2/storage': 'angularfire2.storage',
-  'angularfire2/messaging': 'angularfire2.messaging',
+  '@angular/fire': 'angularfire2',
```
Comment (sarunint) [inline tools/build.js]: Should these be changed?

Comment (jamesdaniels) [reply tools/build.js]: We've decided to keep the UMD names as they are; not worth breaking anyone using them.

**Comment by Chan4077** [comment]
@davideast Will the `angularfire2` package be marked as deprecated?

And will this repository be renamed to :fire:?

**Comment by davideast** [comment]
@Chan4077 Yes it will be! And why not an emoji 😂? Just kidding! We will likely do a rename soon for the repo.

**Pull request merged**